### PR TITLE
Translate helm command : helm status in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>
Translate helm command : helm status in Korean
- https://helm.sh/docs/helm/helm_status/

### CODE REVIEW

**Comment by bluefriday** [comment]
@singlemocha @jmnote 
리뷰 부탁드립니다.

File: content/ko/docs/helm/helm_status.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Helm Status"
+---
+
+## helm status
+
+명명된 릴리스의 상태를 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:7]:
```suggestion
명명된 릴리스의 상태를 표시한다
```

File: content/ko/docs/helm/helm_status.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### 개요
+
+
+이 명령어는 명명된 릴리스의 상태를 표시한다.
+상태는 다음으로 구성된다:
+- 마지막 배포 시간
+- 릴리스가 있는 k8s 네임스페이스
+- 릴리스 상태(unknown, deployed, uninstalled, superseded, failed, uninstalling, pending-install, pending-upgrade or pending-rollback 의 값이 가능)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:16]:
```suggestion
- 릴리스의 상태 (가능한 값: unknown, deployed, uninstalled, superseded, failed, uninstalling, pending-install, pending-upgrade or pending-rollback)
```

File: content/ko/docs/helm/helm_status.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+이 릴리스로 구성되고 종류별로 정렬된 리소스 목록
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:17]:
```suggestion
이 릴리스를 구성하는, 종류(kind)로 정렬된 리소스 목록
```

File: content/ko/docs/helm/helm_status.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- 해당되는 경우의 마지막 테스트 케이스에 대한 새부 정보
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:18]:
```suggestion
- 해당되는 경우, 마지막 테스트 스위트(test suite) 수행에 관한 세부 정보
```

File: content/ko/docs/helm/helm_status.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+helm status RELEASE_NAME [flags]
+```
+
+### 옵션
+
+```
+  -h, --help            helm status 에 대한 도움말
+  -o, --output format   지정된 형식으로 결과를 표시. 허용되는 값: table, json, yaml (default table)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:30]:
```suggestion
  -o, --output format   지정된 형식으로 결과를 표시한다. 허용되는 값: table, json, yaml (기본값 table)
```

File: content/ko/docs/helm/helm_status.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+      --revision int    설정되었을경우, 배포판과 함께 명명된 릴리스의 상태를 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:31]:
```suggestion
      --revision int    지정하면, 해당 리비전의 명명된 릴리스 상태를 표시한다
```

File: content/ko/docs/helm/helm_status.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+### 부모 명령어에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:37]:
```suggestion
      --add-dir-header                   이 값이 true이면, 헤더에 파일 디렉토리를 추가
```

File: content/ko/docs/helm/helm_status.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:38]:
```suggestion
      --alsologtostderr                  파일처럼 표준 오류로도 로그 출력
```

File: content/ko/docs/helm/helm_status.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+      --debug                            장황한(verbose) 출력 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_status.md:48]:
```suggestion
      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다

**Comment by bluefriday** [comment]
> 수고하셨습니다

@jmnote 리뷰 감사합니다. 의견 주신 부분 반영하였습니다.

**Comment by jmnote** [comment]
lgtm

**Comment by singlemocha** [comment]
LGTM

File: content/ko/docs/helm/helm_status.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일 대신 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_status.md:51]: Same concern as the other PR: why is this default changing?

**Comment by bridgetkromhout** [comment]
Everything else in this PR seems fine, but I am hoping to understand if the change I flagged for the default `--repository-cache string` is intended or perhaps is just a local artifact?

File: content/ko/docs/helm/helm_status.md (hunk 10 above)
Comment (singlemocha) [reply content/ko/docs/helm/helm_status.md:51]: I believe 'default' in these strings are referring default value, not a command.  If this is true, we need to translate it.

Comment (singlemocha) [reply content/ko/docs/helm/helm_status.md:51]: Sorry, I misunderstood your comment.  Fix is needed and default should match to English.  Should change as below;
캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "default "~/.cache/helm/repository")

Comment (bluefriday) [reply content/ko/docs/helm/helm_status.md:51]: I will modify every I PRed as your comment like issue #810 :D

**Comment by bluefriday** [comment]
> Everything else in this PR seems fine, but I am hoping to understand if the change I flagged for the default `--repository-cache string` is intended or perhaps is just a local artifact?

@bridgetkromhout Thanks for you review :D

**Review by bridgetkromhout — APPROVED** [review]
LGTM!

**Pull request merged**